internal/impl/nats/input.go

```diff
@@ -33,6 +33,7 @@ You can access these metadata fields using [function interpolation](/docs/config
 ` + auth.Description()).
 		Field(service.NewStringListField("urls").
 			Description("A list of URLs to connect to. If an item of the list contains commas it will be expanded into multiple URLs.").
+			Default([]string{"nats://127.0.0.1:4222"}).
```

Comment:
Generally I avoid having defaults for stuff like urls and addresses, we used to have them all over the shop and people kept getting bitten when forgetting to specify them explicitly.